appscale/tools/appcontroller_client.py

```diff
@@ -461,32 +551,43 @@ def create_user(self, username, password, account_type='xmpp_user'):
       password: A sha1-hashed password that is bound to the given username.
       account_type: A str that indicates if this account can be logged into by
         XMPP users.
+    Raises:
+      AppControllerException if unable to create user.
     """
     AppScaleLogger.log("Creating new user account {0}".format(username))
     while 1:
       try:
-        result = self.run_with_timeout(self.LONGER_TIMEOUT,
-          'Request to create user timed out.', self.DEFAULT_NUM_RETRIES,
-          self.server.create_user, username, password, account_type, self.secret)
+        result = self.run_with_timeout(
+          self.LONGER_TIMEOUT, self.DEFAULT_NUM_RETRIES,
+          self.server.create_user, username, password, account_type,
+          self.secret)
         break
-      except Exception, exception:
+      except Exception as exception:
         AppScaleLogger.log("Exception when creating user: {0}".format(exception))
```

Comment:
Should this catch the timeout exception and break the loop?

Reply:
I think my concern was mainly due to the possibility of unbounded retries rather than the handling of timeouts specifically.

Reply:
I agree that we probably don't want infinite retries. I'll open another PR after this one is merged to give those loops a limit.